components/experimental/src/dimension/provider/currency_compact.rs

```diff
@@ -33,11 +33,9 @@ pub struct ShortCurrencyCompactV1<'data> {
     ///         `"1000-count-one-alt-alphaNextToNumber": "¤ 0K"`
     ///                 -> key1 = 3, key2 = CompactCount::OneAlt, value = "¤ 0K"
     #[cfg_attr(feature = "serde", serde(borrow))]
-    pub compact_patterns: ZeroMap2d<'data, i8, CompactCount, str>,
+    pub compact_patterns: ZeroMap<'data, (i8, CompactCount), str>,
```

Comment:
you have 4 bits left in `CompactCountULE`, what's the range of the log10?

Reply:
there is no limit for log10, but I think to put the limit `16` is kinda a low limit

Reply:
yeah it'd be -8 to 7 as well.